lib/ui/plugins.dart

```diff
@@ -4,31 +4,42 @@
 
 part of dart.ui;
 
-/// An wrapper for a raw callback handle.
+/// A wrapper for a raw callback handle.
+///
+/// This is the return type for [PluginUtilities.getCallbackHandle].
 class CallbackHandle {
-  final int _handle;
-
   /// Create an instance using a raw callback handle.
   ///
   /// Only values produced by a call to [CallbackHandle.toRawHandle] should be
   /// used, otherwise this object will be an invalid handle.
   CallbackHandle.fromRawHandle(this._handle)
       : assert(_handle != null, "'_handle' must not be null.");
 
-  /// Get the raw callback handle to pass over a [MethodChannel] or isolate
-  /// port.
+  final int _handle;
+
+  /// Get the raw callback handle to pass over a [MethodChannel] or [SendPort]
+  /// (to pass to another [Isolate]).
   int toRawHandle() => _handle;
 
   @override
-  int get hashCode => _handle;
+  bool operator ==(dynamic other) {
+    if (runtimeType != other.runtimeType)
```

Comment:
Any particular reason for this change here? Does this not accomplish the same thing?